integrations/vault/auths/approle/src/main/java/io/helidon/integrations/vault/auths/approle/AppRoleVaultAuth.java

```diff
@@ -68,15 +69,17 @@ public static Builder builder() {
 
     @Override
     public Optional<RestApi> authenticate(Config config, Vault.Builder vaultBuilder) {
-        boolean enabled = config.get("auth.app-role.enabled").asBoolean().orElse(true);
+        boolean enabled = config.get("auth.app-role.enabled")
+                                .asBoolean()
+                                .orElse(true);
 
         if (!enabled) {
             return Optional.empty();
         }
         Optional<String> maybeAppRoleId = Optional.ofNullable(this.appRoleId)
-                .or(() -> config.get("auth.app-role.role-id")
-                        .asString()
-                        .asOptional());
+                                                  .or(() -> config.get("auth.app-role.role-id")
```

Comment:
I think you have a different configuration of formatting than the one in `etc` in the project. Could you validate it? I am not a big fan of reformatting statements such as this one, as on the next PR by somebody else, it may be reformatted back

Reply:
this is a general comment to everything in this PR

Reply:
Have used idea formatter provided in slack and used that across PR.